Doc/library/venv.rst

```diff
@@ -81,21 +83,6 @@ It also creates a :file:`lib/pythonX.Y/site-packages` subdirectory
 (on Windows, this is :file:`Lib\site-packages`).
 If an existing directory is specified, it will be re-used.
 
-.. versionchanged:: 3.5
-   The use of ``venv`` is now recommended for creating virtual environments.
-
-.. deprecated-removed:: 3.6 3.8
-   :program:`pyvenv` was the recommended tool for creating virtual environments
-   for Python 3.3 and 3.4, and replaced in 3.5 by executing ``venv`` directly.
-
-.. highlight:: none
-
-On Windows, invoke the ``venv`` command as follows:
-
-.. code-block:: ps1con
-
-   PS> python -m venv C:\path\to\new\virtual\environment
-
 The command, if run with ``-h``, will show the available options::
 
    usage: venv [-h] [--system-site-packages] [--symlinks | --copies] [--clear]
```

Comment:
@Mr-Sunglasses I understood that the problem is that there is duplicate information about `venv` cli (you didn't replace it, you just added a new one, check these lines and below)

could you remove it leaving only the use of `.. option::` directives?

Reply:
You are absolutely right. I forgot about it.

Reply:
In this case, I suggest shortening the help. Currently it's as follows:

![image](https://github.com/user-attachments/assets/1ba88499-3136-4e65-8bc8-423cd2c72592)

Instead I suggest we only keep 

```text
usage: venv [-h] [--system-site-packages] [--symlinks | --copies] [--clear]
            [--upgrade] [--without-pip] [--prompt PROMPT] [--upgrade-deps]
            [--without-scm-ignore-files]
            ENV_DIR [ENV_DIR ...]

Creates virtual Python environments in one or more target directories.
...
Once an environment has been created, you may wish to activate it, e.g. by
sourcing an activate script in its bin directory.
```

Reply:
and maybe it would be better to remove following text from code block:
```
Creates virtual Python environments in one or more target directories.
...
Once an environment has been created, you may wish to activate it, e.g. by
sourcing an activate script in its bin directory.
```